src/status_im2/contexts/quo_preview/wallet/account_card.cljs

```diff
@@ -0,0 +1,129 @@
+(ns status-im2.contexts.quo-preview.wallet.account-card
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]
+            [quo2.core :as quo]
+            [quo2.components.icon :as icon]
+            [reagent.core :as reagent]
+            [utils.collection]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "Type:"
+    :key     :type
+    :type    :select
+    :options [{:key   :default
+               :value "Default"}
+              {:key   :watch-only
+               :value "Watch Only"}
+              {:key   :add-account
+               :value "Add Account"}
+              {:key   :empty
+               :value "Empty"}
+             ]}
+   {:label   "Customization color:"
+    :key     :customization-color
+    :type    :select
+    :options (map (fn [[color-kw _]]
+                    {:key   color-kw
+                     :value (name color-kw)})
+                  colors/customization)}
+   {:label "Name:"
+    :key   :name
+    :type  :text}
+   {:label "Balance:"
+    :key   :balance
+    :type  :text}
+   {:label "Percentage value:"
+    :key   :percentage-value
+    :type  :text}
+   {:label "Amount:"
+    :key   :amount
+    :type  :text}
+   {:label "Metrics:"
+    :key   :metrics?
+    :type  :boolean}
+   {:label "Loading:"
+    :key   :loading?
+    :type  :boolean}
+   {:label "Emoji:"
+    :key   :emoji
+    :type  :text}])
+
+(defn initial-state
+  [type]
+  (case type
+    :default
+    {:name                "Alisher account"
+     :balance             "€2,269.12"
+     :percentage-value    "16.9%"
+     :amount              "€570.24"
+     :customization-color :purple
+     :type                :default
+     :emoji               "💎"}
+
+    :empty
+    {:name                "Account 1"
+     :balance             "€0.00"
+     :percentage-value    "0.00"
+     :amount              "0.00"
+     :customization-color :blue
+     :type                :empty
+     :emoji               "🔎"}
+
+    :watch-only
+    {:name                "Wach-only"
+     :balance             "€2,269.12"
+     :percentage-value    "0.00"
+     :amount              "0.00"
+     :customization-color :blue
+     :type                :watch-only
+     :emoji               "🔎"}
+
+    :add-account
+    {:customization-color :blue
+     :type                :add-account}))
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom (initial-state :default))]
+    [:f>
+    (fn []
+      (rn/use-effect
+        (fn [] (reset! state (initial-state (:type @state))))
+        [(:type @state)])
+      [rn/view
+       {:style {:flex 1}}
+       [rn/view
+        {:style {:margin-vertical 40
+                 :padding-left    40
+                 :flex-direction  :row
+                 :align-items     :center}}
+        [text/text
+         {:size   :heading-1
+          :weight :semi-bold} "Account card"]
```

Comment:
Missing line break after the closing bracket. https://guide.clojure.style/#vertically-align-fn-args